### ISSUE TITLE
New format for calib tables

### ISSUE THREAD

**Issue by bformet** [cguilloud/xcalibu#3]
As discussed I will propose a simpler format for calib tables, to ease life of scientists when writing tables by hand.

Something like this:

```
# comments still start with #

# headers will be optional with reasonable default value
CALIB_NAME = Motor
CALIB_TYPE = TABLE
CALIB_TIME = 1400081171.300155
CALIB_DESC = "test simple  table"

#Energy Motor
4.93990 -0.146702
5.00000 -0.145901
5.50000 -0.141491
5.75030 -0.138115
5.99960 -0.136984
6.50000 -0.133978
6.99990 -0.13299
```

Headers will be not mandatory
* default type will be TABLE
* name will be left empty or will match file name ?
* headers will be written when saving file

~~I don't like too much the idea to support multiple file format (old and new).~~
~~But I will implement a conversion utility (to be executed manually).~~
edit: both file formats supported for loading and saving

@cguilloud do you agree with this?

edit:

- no changes to POLY calib type
- CALIB_TIME being epoch is also not very human friendly, could be changed to string date ?

**Comment by bformet** [comment]
It would be interesting to preserve comments when saving

**Comment by bformet** [comment]
closed by #4 & #5
